lib/算經.wy

```diff
@@ -182,10 +183,12 @@
 	若非。乃得陽也。
 是謂「不可算數乎」之術也。
 
+疏曰「「下溢。同犀之Math.floor也。」」
 吾有一術。名之曰「下溢」。欲行是術。必先得一數。曰「符」。乃行是術曰。
 	乘「符」以「微位冪」。乘其以「至微數」。乃得矣。
 是謂「下溢」之術也。
 
+疏曰「「下溢。同犀之Math.ceil也。」」
 吾有一術。名之曰「上溢」。欲行是術。必先得一數。曰「符」。乃行是術曰。
```

Comment:
This is a non-export function to generate floating point overflow, not Math.ceil